Examples/MAX32650/StaticLibrary/example_lib/example_lib.mk

```diff
@@ -0,0 +1,31 @@
+# Example static library Makefile
+
+# **********************************************************
+# Set the name of your library here. It will be built as CUSTOM_LIB_NAME_softfp.a,
+# LIB_NAME_hardfp.a or just LIB_NAME.a under the project build directory.
+CUSTOM_LIB_NAME=libExample
+# Insert your source files below
+CUSTOM_LIB_SRC_FILES = example.c
```

Comment:
It will be nice if src and include related things be moved out of this file. It will simplify usage of it.